### PR TITLE
Add flag for fuji

### PR DESCRIPTION
**Pull request by holisticode**
Closes https://github.com/ava-labs/avalanche-cli/issues/121

### CODE REVIEW

File: cmd/subnetcmd/deploy.go [hunk 1]
```diff
@@ -90,8 +90,15 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 
 	// get the network to deploy to
 	var network models.Network
-	if deployLocal {
-		network = models.Local
+	if deployNetwork != "" {
+		switch deployNetwork {
+		case strings.ToLower(models.Fuji.String()):
+			network = models.Fuji
+		case "local":
```
Comment (felipemadero) [inline cmd/subnetcmd/deploy.go:97]: there is also a modes.Local.String()

Comment (holisticode) [reply cmd/subnetcmd/deploy.go:97]: Yes but that `String()` function prints `Local Network`. I think we just want people to enter `local`.

File: cmd/subnetcmd/deploy.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ subnet and deploy it on Fuji or Mainnet.`,
 		RunE:         deploySubnet,
 		Args:         cobra.ExactArgs(1),
 	}
-	cmd.Flags().BoolVarP(&deployLocal, "local", "l", false, "deploy to a local network")
+	cmd.Flags().StringVarP(&deployNetwork, "network", "n", "", "network to which to deploy to (one of 'local' or 'fuji')")
```
Comment (dasconnor) [inline cmd/subnetcmd/deploy.go:49]: Oh interesting. I was actually thinking we would do `--local`, `--fuji`, and `--mainnet` and then just check that only one flag was selected. We need to use this pattern a few places, so we should make a global decision. Does anyone have a strong opinion? @holisticode @felipemadero

File: cmd/subnetcmd/deploy.go (hunk 1 above)
Comment (dasconnor) [reply cmd/subnetcmd/deploy.go:97]: If we take the approach I mentioned above, we can synchronize this with the model string, which I think would be preferable.

File: cmd/subnetcmd/deploy.go (hunk 2 above)
Comment (holisticode) [reply cmd/subnetcmd/deploy.go:49]: I am suggesting here to use the `--network` flag so I stand to that suggestion. However, no strong opinion.

Comment (holisticode) [reply cmd/subnetcmd/deploy.go:49]: Ok refactored to use dedicated flags

File: cmd/subnetcmd/deploy.go (hunk 1 above)
Comment (holisticode) [reply cmd/subnetcmd/deploy.go:97]: With bool flags it's not even needed

File: cmd/subnetcmd/deploy.go (hunk 2 above)
Comment (felipemadero) [reply cmd/subnetcmd/deploy.go]: No strong opinion

**Review by felipemadero — COMMENTED** [review]
LGTM

**Review by dasconnor — APPROVED** [review]
LGTM

**Pull request merged**